drivers/wifi/nrf700x/src/net_if.c

```diff
@@ -307,6 +310,14 @@ int nrf_wifi_if_send(const struct device *dev,
 		goto unlock;
 	}
 
+	def_dev_ctx = wifi_dev_priv(rpu_ctx_zep->rpu_ctx);
+	net_buff_check = net_pkt_to_nbuf(pkt);
+	if (!net_buff_check) {
+		LOG_DBG("Failed to allocate net_pkt");
+		host_stats->total_tx_drop_pkts++;
+		goto out;
+	}
```

Comment:
Doesn't `net_buff_check` need to get used further (assuming it was allocated successfully above) ?